odoo/addons/base/models/ir_model.py

```diff
@@ -2199,7 +2193,6 @@ def delete(records):
 
         # remove constraints
         constraints = self.env['ir.model.constraint'].search([('module', 'in', modules.ids)])
-        constraints._module_data_uninstall()
```

Comment:
Looks like `constraints` is no longer useful, so you can remove the first line as well.